ur5_e_moveit_config/config/ur5e.srdf

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" ?>
+<!--This does not replace URDF, and is not an extension of URDF.
+    This is a format for representing semantic information about the robot structure.
+    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
+-->
+<robot name="ur5e">
```

Comment:
Should this also be with "_"?

```suggestion
<robot name="ur5_e">
```

Reply:
[Here](https://github.com/ros-industrial/universal_robot/blob/e8234318cc94dd287617fc196c1b32bf6e4074a9/ur5_e_moveit_config/config/ur5e.srdf#L6) it is stated as ur5e.